### PR TITLE
Add flag options for flush and offline modes

### LINKED ISSUE #65: Add command to Empty pics folder
often the pics folder get's stale and then we get the same images repeatedly its best if there's a command to like `python3 UIP.py --flush` to delete all the pics in the pics folder

### PR DESCRIPTION
**Pull request by nkprince007**
Fixes #65

### CODE REVIEW

File: UIP.py [hunk 1]
```diff
@@ -1,16 +1,33 @@
-import sys
+import sys, argparse, os, shutil
+from uiplib.constants import CURR_DIR, PICS_FOLDER
 from uiplib.scheduler import scheduler
 
 if __name__ == "__main__":
     print("Hey this is UIP! you can use it to download"
           " images from reddit and also to schedule the setting of these"
           " images as your desktop wallpaper.")
+    parser = argparse.ArgumentParser()
```
Comment (abhsag24) [inline UIP.py]: awesome :D

File: UIP.py [hunk 2]
```diff
@@ -1,16 +1,33 @@
...
+    parser.add_argument("--offline", action="store_true",
+                        help="Runs UIP in offline mode.")
+    parser.add_argument("--flush", action="store_true",
+                        help="Delete all downloaded wallpapers"
+                             " and downloads new ones. "
+                             "When combined with --offline,"
+                             " deletes the wallpapers and exits.")
+    args = parser.parse_args()
     try:
         offline = False
```
Comment (abhsag24) [inline UIP.py]: why do we have this variable still? get rid of it!

**Comment by abhsag24** [comment]
@nkprince007 add argparse to requirements.txt

File: UIP.py (hunk 2 above)
Comment (nkprince007) [inline UIP.py]: Ok, bro.

**Comment by nkprince007** [comment]
argparse is standard library, right?

**Comment by abhsag24** [comment]
@nkprince007 dont think so

**Comment by abhsag24** [comment]
ohh it is sorry

**Comment by nkprince007** [comment]
No problem, bro :)

**Comment by abhsag24** [comment]
@nkprince007 can you check gitter :D

**Comment by abhsag24** [comment]
and yes just a small change about the offline variable required :P

**Comment by nkprince007** [comment]
Is it okay now @abhsag24

File: UIP.py [hunk 3]
```diff
@@ -1,18 +1,34 @@
...
+                             " and downloads new ones. "
+                             "When combined with --offline,"
+                             " deletes the wallpapers and exits.")
+    args = parser.parse_args()
     try:
-        offline = False
-        if len(sys.argv) > 1 and str(sys.argv[1]) == '--offline':
+        if args.offline:
             print("You have choosen to run UIP in offline mode.")
             offline = True
```
Comment (abhsag24) [inline UIP.py]: still missed here :P

File: UIP.py [hunk 4]
```diff
@@ -1,18 +1,33 @@
...
 
 if __name__ == "__main__":
     print("Hey this is UIP! you can use it to download"
           " images from reddit and also to schedule the setting of these"
           " images as your desktop wallpaper.")
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--offline", action="store_true",
+                        help="Runs UIP in offline mode.")
+    parser.add_argument("--flush", action="store_true",
+                        help="Delete all downloaded wallpapers"
```
Comment (abhsag24) [inline UIP.py]: loves this :P

**Pull request merged**